packages/devtools_shared/lib/src/deeplink/deeplink_manager.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:shelf/shelf.dart' as shelf;
+
+import '../server/server_api.dart';
+
+class DeeplinkManager {
+  static final _buildVariantJsonRegex = RegExp(r'(\[.*\])');
```

Comment:
Not sure if there is a better way to run and parse flutter command. Please advice